src/net.cpp

```diff
@@ -551,7 +551,6 @@ void CNode::copyStats(CNodeStats &stats, const std::vector<bool> &m_asmap)
     X(m_legacyWhitelisted);
     X(m_permissionFlags);
     if (m_tx_relay != nullptr) {
-        LOCK(m_tx_relay->cs_feeFilter);
         stats.minFeeFilter = m_tx_relay->minFeeFilter;
```

Comment:
nit: Make atomic operations explicit:
```suggestion
        stats.minFeeFilter = m_tx_relay->minFeeFilter.load();
```

Reply:
Going to leave as is to minimize the diff